deps/fast_float_c_interface/Makefile

```
@@ -0,0 +1,15 @@
+CXX?=c++
+# FASTFLOAT_ALLOWS_LEADING_PLUS allows +inf to be parsed as inf, instead of error. 
+CXXFLAGS=-std=c++11 -fPIC -D FASTFLOAT_ALLOWS_LEADING_PLUS
+
+.PHONY: all clean
+all: valkey_strtod.o
+clean:
+	rm -f *.o || true;
```

With `-f`, rm never fails.

```suggestion
	rm -f *.o
```

---

It does on mac. 
```
/tmp % rm -f *.0
zsh: no matches found: *.0
```

---

`-(rm -f *.o)` I guess would be the normal makefile way, to ignore the error.

---

MacOS have ridiculously outdated coreutils, because they refused to lift to GPLv3. I guess it's one of the reasons I'm running fedora on my macbook instead of macOS, hehe.

---

```suggestion
	-(rm -f *.o)
```